src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespacebeforecasedefaultcolon/InputNoWhitespaceBeforeCaseDefaultColon.java

```diff
@@ -0,0 +1,52 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.nowhitespacebeforecasedefaultcolon;
+
+/*
+* Config = default
+*/
+public class InputNoWhitespaceBeforeCaseDefaultColon {
+    {
+        switch(1) {
+            case 1 : // violation
+                break;
+            case 2:
+                break;
+            default : // violation
+                break;
+        }
+
+        switch(2) {
+            case 2:
+                break;
+            default:
+                break;
+
+        }
+
+        switch(3) {
+            case 3 /* space around*/ : // violation
+                break;
+            default/* no space around */:
+                break;
+        }
+
+        switch(4) {
+            case 4
+                    : // violation
+                break;
+            default:
+                switch(5) {
+                    case 5 : // violation
+                        break;
+                    case 6
+: // violation
+                    default:
+                        break;
+                }
+                break;
+        }
+
+         for (int var1 : new int[]{}) {}
+
+         for (int var2: new int[]{}) {}
```

Comment:
if you want to show that this cases should not cause violation, ternary operator can be added here as well.

Reply:
Done.